samples/bluetooth/mesh_provisioner/src/main.c

```diff
@@ -348,7 +349,9 @@ void main(void)
 	printk("Bluetooth initialized\n");
 	bt_ready();
 
+	#if DT_NODE_HAS_STATUS(SW0_NODE, okay)
```

Comment:
Minor: If I'm not mistaken, preprocessor directives should not have indentation.